cmd/checkapi/allowlist.txt

```diff
@@ -7,4 +7,4 @@ receiver/collectdreceiver
 receiver/dockerstatsreceiver
 receiver/journaldreceiver
 receiver/kafkareceiver
```

Comment:
Should we remove kafkareceiver in allowlist.txt?  Then we can run command `go run cmd/checkapi/main.go . ` to confirm again